index.html

```diff
@@ -554,6 +604,772 @@ <h2>OpenID Connect</h2>
+The Multikey data model is also capable of encoding secret keys, whose subtypes
+include <em>symmetric keys</em> and <em>private keys</em>.
+            </p>
+
+            <pre class="example nohighlight"
+              title="Multikey encoding of a Ed25519 secret key">
+{
+  "@context": ["https://w3id.org/security/suites/secrets/v1"],
+  "id": "did:example:123456789abcdefghi#keys-1",
+  "type": "Multikey",
+  "controller": "did:example:123456789abcdefghi",
+  "secretKeyMultibase": "z3u2fprgdREFtGakrHr6zLyTeTEZtivDnYCPZmcSt16EYCER"
+}
+            </pre>
+
+            <p>
+In the example above, the `secretKeyMultibase` value starts with the letter `z`,
+which is the [[?MULTIBASE]] header that indicates that the binary data is
+base58btc-encoded. The decoded binary data
+[[?MULTICODEC]] header is `0x1300`, which specifies that the remaining data
+is a 32-byte raw Ed25519 private key.
+            </p>
+
+          </section>
+
+          <section>
+            <h3>JsonWebKey</h3>
+            <p>
+The JSON Web Key (JWK) data model is a specific type of <a>verification method</a>
+that uses the JWK specification [[RFC7517]] to encode key types into a
+set of parameters.
+            </p>
+
+            <p>
+When specifing a `JsonWebKey`, the object takes the following form:
+            </p>
+
+            <dl>
+              <dt>type</dt>
+              <dd>
+The value of the `type` property MUST contain the string `JsonWebKey`.
+              </dd>
+              <dt><dfn class="lint-ignore">publicKeyJwk</dfn></dt>
+              <dd>
+The `publicKeyJwk` property is OPTIONAL. If present, its value MUST
+be a <a data-cite="INFRA#ordered-map">map</a> representing a JSON Web Key that
+conforms to [[RFC7517]]. The <a data-cite="INFRA#ordered-map">map</a> MUST NOT
+include any members of the private information class, such as `d`, as described
+in the <a href="https://tools.ietf.org/html/rfc7517#section-8.1.1">JWK
+Registration Template</a>. It is RECOMMENDED that verification methods that use
+JWKs [[RFC7517]] to represent their <a>public keys</a> use the value of `kid` as
+their fragment identifier. It is RECOMMENDED that JWK `kid` values be set to
+the public key fingerprint [[RFC7638]]. See the first key in
+<a href="#example-various-verification-method-types"></a> for an example of a
+public key with a compound key identifier.
+              </dd>
+              <dt><dfn class="lint-ignore">secretKeyJwk</dfn></dt>
+              <dd>
+The `secretKeyJwk` property is OPTIONAL. If present, its value MUST be a <a
+data-cite="INFRA#ordered-map">map</a> representing a JSON Web Key that conforms
+to [[RFC7517]].
+              </dd>
+            </dl>
+
+            <p>
+An example of an object that conforms to this data model is provided below:
+            </p>
+
+            <pre class="example nohighlight"
+              title="JSON Web Key encoding of an Ed25519 public key">
+{
+  "@context": ["https://www.w3.org/ns/security/jwk/v1"],
```

Comment:
```suggestion
```

Reply:
we don't expect to see this nested context in controller documents, which this section is describing.